proxy/pkg/util/util.go

```diff
@@ -50,6 +53,7 @@ var (
 
 	managedLabelList = proxyconfig.GetManagedClusterLabelList()
 	syncLabelList    = proxyconfig.GetSyncLabelList()
+	accessReviewer   *rbac.AccessReviewer
```

Comment:
Can we avoid adding yet another global variable in favor of a local variable that is passed around properly? Particularly a global variable that can be `nil` is very error prone.

Reply:
Fixed. Outside of initialization, this variable is not directly accessed, and is passed around between methods. Further, the type is changed to an interface to mock up for unit testing.